backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/importer/annotations/BooleanField.kt

```diff
@@ -0,0 +1,4 @@
+package xyz.elitese.ehrenamtskarte.importer.annotations
+
+@Target(AnnotationTarget.FIELD)
+annotation class BooleanField
```

Comment:
Minor: Kind of weird that these are required. Shouldn't Klaxon support this?

Reply:
I think the issue is that all fields in the original json are strings, so they need to be converted to bools, ints etc. @aiquita can you confirm that this was the reason?